### PR TITLE
Adding eland package

### PR DESCRIPTION
**Pull request by sethmlarson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by stevedodson** [comment]
Hi! Could I be listed as a maintainer as well?

**Comment by sethmlarson** [comment]
@conda-forge/help-python Is this the proper way to specify a Python package that has no binary dependencies? Still getting all the different OS builds, wondering if those are necessary. :)

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml:1]: Please do not delete this file

File: recipes/eland/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  imports:
+    - eland
+    - eland.ml
+
+about:
+  home: https://eland.readthedocs.io
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/eland/meta.yaml:36]: looks like this should be `license_file: LICENSE.txt`

File: recipes/eland/run_test.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from eland import (
```
Comment (synapticarbors) [inline recipes/eland/run_test.py:1]: I don't think you need the `run_test.py` file, since the import tests in the `meta.yaml` does the same thing

**Comment by synapticarbors** [comment]
@sethmlarson -- the `noarch: python` line makes a single os-independent build, but in `staged-recipes` all of the different CIs still run. Once this goes to the feedstock, only a single build will be produced.

**Comment by sethmlarson** [comment]
@synapticarbors Thanks for the help and review comments! I'm seeing a failure due to `xgboost` not being available on Windows but we won't have a Windows build on our feedstock as it's pure-Python, is that acceptable?

**Comment by synapticarbors** [comment]
Do you actually need the `requires` section with `xgboost` and `scikit-learn` under `test`? The `import` tests are only really doing an existence check and if the library doesn't require them, then you don't need to have the `requires`. Generally that section is used if you were running a test suite the required something like `pytest` to run, but `pytest` wasn't need to use the library in general.

**Comment by sethmlarson** [comment]
@synapticarbors Yeah we require those libraries to import `eland.ml` as they're optional dependencies.

**Comment by synapticarbors** [comment]
ok, that should be fine then.

**Comment by sethmlarson** [comment]
@synapticarbors Thanks much! :sparkles:

**Pull request merged**